src/transformers/benchmark/benchmark.py

```diff
@@ -149,11 +182,22 @@ def inference(self, model_name, batch_size, sequence_length, trace_memory=False)
                 vocab_size, (batch_size, sequence_length), dtype=torch.long, device=self.args.device
             )
 
+            if self.args.torchscript:
+                with torch.no_grad():
+                    if config.is_encoder_decoder:
+                        raise NotImplementedError("Torchscript is currently not supported for EncoderDecoder models")
+                    else:
+                        inference_model = torch.jit.trace(model, input_ids)
```

Comment:
This is more or less all I added for torchscript...is there anything else I should pay attention to?